### DISCUSSION TITLE
Stride

### DISCUSSION THREAD

**Discussion by Chtchou** [facebookresearch/detectron2#3135] [Q&A]
I was wondering how I can choose the stride for your anchor generator.

Yet, when I look at the code it the stride comes from the input_shape(ShapeSpec). But it seems that the input_shape is never given a stride argument anywhere.

So I was wondering If we could control the stride in the anchor generator or is there any other way to use it?

**Comment by yusiyoh** [comment]
Did you manage to find a solution for this? @Chtchou
